x-pack/plugins/ml/server/models/data_recognizer/modules/security_windows/ml/v2_windows_anomalous_process_all_hosts_ecs.json

```diff
@@ -0,0 +1,56 @@
+{
+  "job_type": "anomaly_detector",
+  "description": "This is a new refactored job which works on ECS compatible events across multiple indices. Security: Windows - Looks for processes that are unusual to all Windows hosts. Such unusual processes may indicate execution of unauthorized services, malware, or persistence mechanisms.",
+  "groups": [
+    "security",
+    "endpoint",
+    "event-log",
+    "sysmon",
+    "windows",
+    "winlogbeat",
+    "process"
+  ],
+  "analysis_config": {
+    "bucket_span": "15m",
+    "detectors": [
+      {
+        "detector_description": "rare by \"process.executable\"",
+        "function": "rare",
+        "by_field_name": "process.executable"
+      }
+    ],
+    "influencers": [
```

Comment:
@blaklaybul should we add `process.executable` to the list of influencers?

Reply:
yes, we should add this. We should mimic the existing job config here: https://github.com/elastic/kibana/blob/master/x-pack/plugins/ml/server/models/data_recognizer/modules/siem_auditbeat/ml/linux_anomalous_process_all_hosts_ecs.json

Reply:
et me evaluate this one and what it looks like in the UI - it may have been removed due to length

Reply:
So the reason it was removed is that this field is way too long to display in the influencer pane. In my opinion it detracts from UX b/c the user cannot read the values in order to make a selection.